tuxemon/combat.py

```diff
@@ -81,8 +81,11 @@ def pre_checking(
         monster.status[0].combat_state = combat
         monster.status[0].phase = "pre_checking"
         result_status = monster.status[0].use(target)
-        if result_status["technique"]:
-            technique = result_status["technique"]
+        if result_status.technique:
+            if len(result_status.technique) > 1:
```

Comment:
You could drop the branches and always use random.choice; it would be more concise. Up to you.

Reply:
it makes sense